app/controllers/api/v1/locations_controller.rb

```diff
@@ -0,0 +1,35 @@
+module Api
+  module V1
+    class LocationsController < ApplicationController
+      before_action :set_location, only: [:update, :destroy]
+
+      def create
+        @school = CodeSchool.find(params[:code_school_id])
+        @school.locations.build(location_params)
+        if @school.save
+          render json: @school
+        else
+          render json: { errors: @school.errors.full_messages }
```

Comment:
For all the error cases, can you render a status to the frontend, that they can leverage.  Here is [an example](https://github.com/OperationCode/operationcode_backend/blob/master/app/controllers/api/v1/team_members_controller.rb#L15) of setting a status in an error case.

Reply:
Do you think it's a good idea to rescue from `StandardError`? I remember learning this isn't such a great idea because it rescues from all types of errors.

Reply:
Yes, good question, that is something to consider.  In general, if you know the error(s) that you are targeting, that could be raised, it is [best to specify them](https://robots.thoughtbot.com/rescue-standarderror-not-exception#best-case-scenario).

But if that is not on the table, then rescueing from `StandardError` is [an option](https://robots.thoughtbot.com/rescue-standarderror-not-exception#when-that39s-not-possible).

In your case, for the `create!` and `update!` cases, you could `rescue` from [ActiveRecord::RecordInvalid](http://api.rubyonrails.org/v5.0/classes/ActiveRecord/RecordInvalid.html).  And for `destroy!`, you could potentially `rescue` from `NoMethodError`, or [ActiveRecord::RecordNotDestroyed](http://api.rubyonrails.org/classes/ActiveRecord/RecordNotDestroyed.html).

Your call.